### PR TITLE
Body info description page

### PR DESCRIPTION
**Pull request by hoanguyennph**

### CODE REVIEW

File: VMS/Pages/ActivityInfoPage/Index.razor [hunk 1]
```diff
@@ -4,10 +4,10 @@
     <a href="@Routes.ActivitySearch">Quay lại</a>
     <div class="layout__divider"></div>
     <div class="row layout__body">
-        <div class="col-7 flex-grow-1 layout__body__main" style="height: 90px;">
-            Main body
+        <div class="col-7 flex-grow-1 layout__body__main">
+            <InfoPage></InfoPage>
```
Comment (LongDuongBIT) [inline VMS/Pages/ActivityInfoPage/Index.razor:8]:
```suggestion
            <InfoPage />
```

File: VMS/Pages/ActivityInfoPage/InfoPage.razor [hunk 2]
```diff
@@ -0,0 +1,74 @@
+﻿<div class="InfoPage">
```
Comment (LongDuongBIT) [inline VMS/Pages/ActivityInfoPage/InfoPage.razor:1]: Please rename all class names, class names won't be capitalized.

File: VMS/Pages/ActivityInfoPage/InfoPage.razor [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    <div class="TopPage">
+        <span class="Return">
+            <a href="@Routes.ActivitySearch" class="Return-text">&#60;&#60; Quay lại</a>
```
Comment (LongDuongBIT) [inline VMS/Pages/ActivityInfoPage/InfoPage.razor:4]: There are two "Quay lại" links in this page. Please remove one.

File: VMS/Pages/ActivityInfoPage/InfoPage.razor [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            </ul>
+        </div>
+    </div>
+</div>
+
+@code {
+    private string typeButton = "heart-button1";
+    private string typeI = "favorite_border";
+
+    protected void animateHeart()
```
Comment (LongDuongBIT) [inline VMS/Pages/ActivityInfoPage/InfoPage.razor:61]:
```suggestion
    private void AnimateHeart()
```

File: VMS/Pages/ActivityInfoPage/InfoPage.razor.scss [hunk 5]
```diff
@@ -0,0 +1,163 @@
+﻿@import '../../Resources/scss/common/_variables';
+
+.TopPage {
+    .Return-text {
+        text-decoration: none;
+        font-family: Montserrat;
```
Comment (LongDuongBIT) [inline VMS/Pages/ActivityInfoPage/InfoPage.razor.scss:6]: DO NOT set font-family, please remove this all other places.

File: VMS/Pages/ActivityInfoPage/Index.razor (hunk 1 above)
Comment (hoanguyennph) [reply VMS/Pages/ActivityInfoPage/Index.razor:8]: done

File: VMS/Pages/ActivityInfoPage/InfoPage.razor (hunk 2 above)
Comment (hoanguyennph) [reply VMS/Pages/ActivityInfoPage/InfoPage.razor:1]: done

File: VMS/Pages/ActivityInfoPage/InfoPage.razor (hunk 4 above)
Comment (hoanguyennph) [reply VMS/Pages/ActivityInfoPage/InfoPage.razor:61]: done

File: VMS/Pages/ActivityInfoPage/InfoPage.razor (hunk 3 above)
Comment (hoanguyennph) [reply VMS/Pages/ActivityInfoPage/InfoPage.razor:4]: done

File: VMS/Pages/ActivityInfoPage/InfoPage.razor.scss (hunk 5 above)
Comment (hoanguyennph) [reply VMS/Pages/ActivityInfoPage/InfoPage.razor.scss:6]: done

**Comment by LongDuongBIT** [comment]
@hoanguyennph I have some comments:
- The link "Quay lại" is too near the navbar, please add margin for them.
image

- You put the right-rail at the wrong position, please fix it
image

**Comment by LongDuongBIT** [comment]
@hoanguyennph Please fix my feedback in Slack.

**Review by LongDuongBIT — APPROVED** [review]
LGTM 👍

**Pull request merged**